scheduler/src/cook/mesos/schema.clj

```diff
@@ -221,6 +227,12 @@
     :db/index true
     :db.alter/_attribute :db.part/db}])
 
+(def migration-port-is-a-count
```

Comment:
In an earlier commit, the port schema in the DB was meant to be a set of the ports allocated. Now, it's the # of ports allocated.